image/templates/helm/shared/templates/02-scanner-06-deployment.yaml.htpl

```diff
@@ -41,6 +41,9 @@ spec:
       tolerations:
         {{- toYaml ._rox.scanner.tolerations | nindent 8 }}
       {{- end }}
+      {{- if ._rox.scanner.hostAliases }}
+      hostAliases: {{ toYaml ._rox.scanner.hostAliases | nindent 8 }}
+      {{- end }}
```

Comment:
I think you'll have to move that to the next line.
```suggestion
      {{- if ._rox.scanner.hostAliases }}
      hostAliases:
        {{ toYaml ._rox.scanner.hostAliases | nindent 8 }}
      {{- end }}
```

Reply:
@johannes94 not with `nindent` (as opposed to `indent`) - it adds a newline